packages/rocketchat-livechat/.app/imports/client/visitor.js

```diff
@@ -63,10 +64,11 @@ export default {
 
 		this.roomSubscribed = roomId;
 
+		const msgTypesNotDisplayed = ['livechat_video_call', 'livechat_navigation_history', 'au'];
 		msgStream.on(roomId, { token: this.getToken() }, (msg) => {
 			if (msg.t === 'command') {
 				Commands[msg.msg] && Commands[msg.msg]();
-			} else if (msg.t !== 'livechat_video_call') {
+            } else if (!_.contains(msgTypesNotDisplayed, msg.t)) {
```

Comment:
no need to use `underscore` here, you can use [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)